assets/js/components/wp-dashboard/WPDashboardPopularPages.js

```diff
@@ -103,15 +106,11 @@ export default function WPDashboardPopularPages( props ) {
 		return <PreviewTable rows={ 6 } />;
 	}
 
-	if ( ! isGatheringData ) {
-		return null;
-	}
```

Comment:
Hmm, odd, yeah, not sure why this is here… 🤔 

Looks like you added it 😆  https://github.com/google/site-kit-wp/commit/53a2f75f4813bf285415f2d35c77a5515db0ed9d#diff-936b7ee65e0faeaa4e88d0a40efb31a526270e87a94b3ec63d28a8abad479f79R102-R104

Anyway, I think it's best to remove 👍🏻